modules/ai/module.php

```diff
@@ -697,8 +693,10 @@ public function ajax_ai_get_image_to_image_outpainting( $data ) {
 		$context = $this->get_request_context( $data );
 		$request_ids = $this->get_request_ids( $data['payload'] );
 		$result = $app->get_image_to_image_out_painting( [
-			'prompt' => $data['payload']['prompt'],
+			'size' => $data['payload']['size'],
+			'position' => $data['payload']['position'],
 			'mask' => $data['payload']['mask'],
+			'image_url' => $data['payload']['image']['url'],
```

Comment:
Please separate to a new version: `get_image_to_image_out_painting_v2`

Reply:
talked f2f, we'll keep it as is